src/DataViews/LiveFunctionsDataViewTest.cpp

```diff
@@ -384,6 +386,34 @@ TEST_F(LiveFunctionsDataViewTest, ContextMenuActionsAreInvoked) {
     CheckExportToCsvIsInvoked(context_menu, app_, view_, expected_contents);
   }
 
+  // Export events to CSV
+  {
+    const uint32_t kThreadId = 111;
+    const std::string kThreadName = "Test Thread";
+    capture_data_->AddOrAssignThreadName(kThreadId, kThreadName);
+    EXPECT_CALL(app_, GetCaptureData).WillRepeatedly(testing::ReturnRef(*capture_data_));
+
+    constexpr uint64_t kStart = 12345;
+    constexpr uint64_t kEnd = 67890;
+    TimerInfo timer;
+    timer.set_start(kStart);
+    timer.set_end(kEnd);
+    timer.set_thread_id(kThreadId);
+    std::vector<const TimerInfo*> timers = {&timer};
+    EXPECT_CALL(app_, GetAllFunctionCalls).WillRepeatedly(testing::Return(timers));
+
+    std::string expected_contents =
+        absl::StrFormat(R"#("Name","Thread","Start","End","Duration (ns)")#"
+                        "\r\n"
+                        R"("%s","%s","%s","%s","%s")"
+                        "\r\n",
+                        kPrettyNames[0], absl::StrFormat("%s [%lu]", kThreadName, kThreadId),
+                        absl::StrFormat("%lu", kStart), absl::StrFormat("%lu", kEnd),
+                        absl::StrFormat("%lu", kEnd - kStart));
```

Comment:
These nested absl::StrFormat are not really readable and unnecessary. You can do `R"("%s","%s [%lu]","%lu","%lu","%lu")" directly. 

In case you will add more timer, maybe its more readable to split it up like `expected_contents = header_line + line_1 + line_2 + ...`

Reply:
Ah, make sense! I was stupid 😂  Thanks for the suggestion!